### DISCUSSION TITLE
set belloff=all doesn't work

### DISCUSSION THREAD

**Discussion by hpcx82** [JetBrains/ideavim#502] [Q&A]
It is quite annoying that it beeps everything I press `Esc`,  and `set belloff=all` seems like doesn't work, is there an effective way to mute all beeps?

**Comment by citizenmatt** [comment]
You use `:set visualbell` which disables the audible bell for errors. The idea is that it shows a "visual" bell, such as flashing the screen, but I don't think any GUI apps support this - IdeaVim doesn't.
